recipes/superlu_dist/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+WORK=$PWD
+# run full build & install twice, once for static, once for shared
+# because it's the cmake way
+for shared in OFF ON; do
+    cd "$WORK"
+    mkdir build_$shared
+    cd build_$shared
+    # enable_blaslib=OFF so OpenBLAS will be found instead of the built-in BLAS
+
+    # build & install once for static
+    cmake .. \
+        -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+        -DCMAKE_INSTALL_LIBDIR="${PREFIX}/lib" \
+        -DCMAKE_C_FLAGS="${CFLAGS} -std=c99 -fPIC" \
```

Comment:
Don't do this. Instead append the flags `-std=c99 -fPIC` to the CFLAGS env variable.